### ISSUE TITLE
when reset custom query  the from change to  90 ,why

### ISSUE THREAD

**Issue by spritecn** [cars10/elasticvue#95]
when reset custom query  the from change  90 ,why
``` json
{
  "query": {
    "query_string": {
      "query": "*"
    }
  },
  "size": 10,
  "from": 90,
  "sort": []
}
```

most man can not  see it
but not result

**Comment by cars10** [comment]
If i understand you correctly: you think it is confusing that "reset query" only resets the actual "query" part of the search body and not the whole search body? 
What do you think about two buttons like this?
![image](https://user-images.githubusercontent.com/4118628/156426982-f81894d3-54b5-4ed8-be82-18ec62def1dc.png)

**Comment by cars10** [comment]
Fixed in `0.39.0`

**Comment by spritecn** [comment]
thx
